src/schema/core.yaml

```diff
@@ -126,6 +127,68 @@ enums:
       liquid:
       gas:
 
+  ChemicalEntityEnum:
+    permissible_values:
+      acetonitrile:
+        meaning: CHEBI:38472
+      alphaLP:
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of threonine, alanine, serine, and valine.
```

Comment:
Please choose a meaning for this permissible value. I recommend one of the IDs on this Wikipedia page, preferably the EC number since it is machine actionable and in the public domain:  https://en.wikipedia.org/wiki/Alpha-lytic_endopeptidase